python/samples/04.ai.a.teamsChefBot/README.md

```diff
@@ -0,0 +1,16 @@
+# Microsoft Teams Conversational Bot with AI: Teams Chef
+
```

Comment:
Can you please make this REAME match the other samples and add at least one screenshot

Reply:
matches the JS sample and added the screenshot